config/routes.rb

```diff
@@ -83,6 +83,8 @@
   }
 
   get '/users/:username', to: redirect('/@%{username}'), constraints: lambda { |req| req.format.nil? || req.format.html? }
+  get '/users/:username/following', to: redirect('/@%{username}/following'), constraints: lambda { |req| req.format.nil? || req.format.html? }, as: :short_account_following
```

Comment:
I don't understand the point of the alias, especially since it's *not* the short version of the path

Reply:
@Gargron True, refactoring this ASAP!